starter-kits/vue-cli-demo/vue.config.js

```diff
@@ -0,0 +1,32 @@
+const HtmlSystemJSHtmlPlugins = require("./plugins/WebpackSystemJSHtmlPlugins");
+module.exports = {
+  css: {
+    extract: true
+  },
+  devServer: {
+    headers: {
+      "Access-Control-Allow-Origin": "*"
+    }
+  },
+  chainWebpack: (config) => {
+    config.output
+      .filename("js/[name].js")
+      .library("vueApp")
+      .libraryTarget("umd")
```

Comment:
Using libraryTarget `"system"` is usually preferred, since it doesn't require the amd.js extra and has support for live bindings.

Reply:
thanks, i realize this project logic  was error few days ago 😂. when i use systemjs and read core document.😐 I will change that and make it correct.